irr/src/CIrrDeviceSDL.h

```diff
@@ -109,6 +109,14 @@ class CIrrDeviceSDL : public CIrrDeviceStub
 		return EIDT_SDL;
 	}
 
+	//! Get the SDL version
+	std::string getVersionString() const override
+	{
+		SDL_version ver;
+		SDL_GetVersion(&ver); // TODO: SDL3 returns this directly instead of using a pointer
```

Comment:
we don't support sdl3 so I don't think this TODO makes sense

Reply:
[a8071c2](https://github.com/luanti-org/luanti/pull/16046/commits/a8071c23917aa76f806042af9d40cf367843aefd). I thought it might be helpful if we _eventually_ move to SDL3, but 🤷